packages/webdriverio/src/commands/element/isDisplayed.js

```diff
@@ -70,7 +70,7 @@ export default async function isDisplayed() {
      * no longer dictates it. In those instances, we pass the element through a script
      * that was provided by Brian Burg of safaridriver.
      */
-    return browser.isW3C && !browser.isMobile && noW3CEndpoint.includes(browser.capabilities.browserName.toLowerCase()) ?
+    return browser.isW3C && browser.capabilities.browserName && noW3CEndpoint.includes(browser.capabilities.browserName.toLowerCase()) ?
```

Comment:
> I also removed the mobile check because new chrome versions on Android also need this fix, instead we now only check for if a browserName is provided

Reply:
how about hybrid apps when there is a need to switch to webview?

Reply:
That wasn't part of this and also not part of the previous scope of the PR. I already had some comments about that on the previous PR (https://github.com/webdriverio/webdriverio/pull/3601)
We need to come up with a solid approach for that in a later stage

Reply:
I suggest changing it to `!isIOS` as it seems that it reflects exactly what we want

Reply:
> I suggest changing it to `!isIOS` as it seems that it reflects exactly what we want

Nope, they also removed the endpoint

Reply:
> I suggest changing it to `!isIOS` as it seems that it reflects exactly what we want

Nope, Safari also removed the endpoint

Reply:
so, condition will pass for Appium with browserName, but for Appium with app. Doesn't make sense for me

Reply:
> so, condition will pass for Appium with browserName, but for Appium with app. Doesn't make sense for me

As said, this decision has already been made with the previous PR, the discussion was already there

See also 
![image](https://user-images.githubusercontent.com/11979740/57233538-1b8c7e00-701f-11e9-8cd3-5bc670b5293a.png)

For now we will support Chrome on Android with Appium, but need to come up with a fix for hybrid apps, like for example check the context, and then determine the method. I don't believe that it will blocking for now for merging this PR

Reply:
I'm worried because I'm testing hybrid app, it uses chrome 73 and test will start failing for me. Am I right?

Reply:
Nope, only with 74

Reply:
sounds good for now then.